apps/nextra/pages/en/build/move/compiler_v2.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: "Compiler v2 (beta)"
+---
+
+# Move Compiler v2 (beta)
+
+Move compiler v2 is a new tool which translates Move source code into Move bytecode. It unifies the architectures of the Move Compiler and the Move Prover, enabling faster innovation in the Move language. It also offers new for defining code optimizations which can be leveraged to generate more gas efficient code for Move programs.
```

Comment:
Could we use either "Move compiler" or "Move Compiler" consistently?

Reply:
Done